src/test/java/org/apache/maven/plugins/dependency/TestGetMojo.java

```diff
@@ -96,6 +117,27 @@ public void testRemoteRepositories()
         mojo.execute();
     }
 
+    /**
+     * Test remote repositories parameter with basic authentication
+     *
+     * @throws Exception in case of errors
+     */
+    public void testRemoteRepositoriesAuthentication()
+        throws Exception
+    {
+        org.eclipse.jetty.server.Server server = createServer( 8000 );
```

Comment:
A concrete prort cannot be. We use the wildcard port `0.0.0.0` and the client connects to the server's local port.

Reply:
I start the server now with a random port, and use server.getURI() to generate the remoteRepositories parameter.